tensorflow_addons/text/crf.py

```diff
@@ -95,22 +116,19 @@ def crf_multitag_sequence_score(inputs, tag_bitmap, sequence_lengths,
 
     # If max_seq_len is 1, we skip the score calculation and simply gather the
     # unary potentials of all active tags.
+    filtered_inputs = crf_filtered_inputs(inputs, tag_bitmap)
```

Comment:
Where are you getting `tag_bitmap` from?

Reply:
You need to look at the signature of `crf_multitag_sequence_score` it is passed as an argument.

Reply:
It is part of the signature of `crf_multitag_sequence_score`

Reply:
My bad, the code above this line was cut off by GitHub. I thought this was part of `crf_sequence_score`.

Reply:
Thanks